ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainActivity.kt

```diff
@@ -90,22 +133,29 @@ class MainActivity : ComponentActivity() {
         copyFilesToExternalDir()
         startForegroundService(Intent(this, MainService::class.java))
 
-        val start = findViewById<Button>(R.id.start)
-        start.setOnClickListener(View.OnClickListener {
-            val path = getExternalFilesDir(null)!!.absolutePath
-            val model = File("$path/mobilenet_v1_1.0_224_quant.tflite")
-            val filter = "other/tensor,format=static,dimension=(string)3:224:224:1,type=uint8,framerate=0/1 ! " +
-                    "tensor_filter framework=tensorflow-lite model=" + model.getAbsolutePath() + " ! " +
-                    "other/tensor,format=static,dimension=(string)1001:1,type=uint8,framerate=0/1"
-            val serverPort = mService?.startServer(filter)
-            val portTextView = findViewById<TextView>(R.id.port)
-            portTextView.text = "Listening on port: " + serverPort.toString();
-        })
-
-        val stop = findViewById<Button>(R.id.stop)
-        stop.setOnClickListener(View.OnClickListener {
-            mService?.stopServer()
-        })
+        // todo: Use database instead of just ArrayList
+        val modelList = ArrayList<ModelInfo>()
+        val path = getExternalFilesDir(null)!!.absolutePath
+
+        val mobileNet = File("$path/mobilenet_v1_1.0_224_quant.tflite")
+        val filter = "other/tensor,format=static,dimension=(string)3:224:224:1,type=uint8,framerate=0/1 ! " +
```

Comment:
For the consistency of variable names, how about
```suggestion
        val mobinetFilter = "other/tensor,format=static,dimension=(string)3:224:224:1,type=uint8,framerate=0/1 ! " +
```

Reply:
Updated! Thank you :)